oauth-server/src/main/java/io/strimzi/kafka/oauth/server/JaasServerOauthValidatorCallbackHandler.java

```diff
@@ -396,6 +388,22 @@ private void validateConfig() {
         }
     }
 
+    private void validateIssuerUri(String validIssuerUri) {
+        if (validIssuerUri == null && config.getValueAsBoolean(ServerConfig.OAUTH_CHECK_ISSUER, true)) {
+            throw new RuntimeException("OAuth validator configuration error: OAUTH_VALID_ISSUER_URI must be set or OAUTH_CHECK_ISSUER has to be set to 'false'");
```

Comment:
Is there nothing more specific than `RuntimeException`?

Reply:
This is a one time configuration check that happens when Kafka Broker is started. If it fails, the Kafka Broker fails to start and there is nothing else to be done, but to fix the configuration. Thus, there is no code, that would use the type of exception as a discriminator for what action to perform next.

For input checking cases IllegalArgumentException is often used but that suggests a specific method argument is the problem. Here it's the content of the configuration file that is the problem.
 
Both server callback handlers use RuntimeException to report configuration errors.

I don't see an obviously better option to use.

Reply:
OK, fair enough.